rules/xcodeproj.bzl

```diff
@@ -61,7 +61,7 @@ def _xcodeproj_aspect_impl(target, ctx):
                 direct_srcs = [],
             ),
         )
-        target_info = _TargetInfo(target = info, targets = depset([info], transitive = _get_attr_values_for_name(deps, _TargetInfo, "targets")))
+        target_info = _TargetInfo(target = info, all_dep_targets = depset([info], transitive = _get_attr_values_for_name(deps, _TargetInfo, "all_dep_targets")))
```

Comment:
im not sure this rename feels very "bazelly". In general, plural fields in a provider as assumed to return a depset, which will contain transitive information. If you did want to clean this up, renaming `target` to `direct_target` would make sense, as other providers use the `direct_*` fields to communicate info about the direct target that is returning that provider